neuralmonkey/config/builder.py

```diff
@@ -26,21 +26,30 @@ def create(self):
 
         class_name = class_parts[-1]
 
-        # TODO should we not assume that everything is from neuralmonkey?
-        module_name = ".".join(["neuralmonkey"] + class_parts[:-1])
-
+        simple_module_path = ".".join(class_parts[:-1])
         try:
-            module = importlib.import_module(module_name)
-        except ImportError as exc:
-            # if the problem is really importing the module
-            if exc.name == module_name:
-                raise Exception(
-                    ("Interpretation '{}' as type name, module '{}' "
-                     "does not exist. Did you mean file './{}'? \n{}")
-                    .format(self.clazz,
-                            module_name, self.clazz, exc)) from None
-            else:
-                raise
+            import tensorflow as tf
```

Comment:
Proč se to importuje zrovna tady? Nebylo by lepší to dát úplně nahoru, když až na výjimky většina jmen tímhle projde?